src/Compiler/Driver/CompilerConfig.fs

```diff
@@ -300,7 +300,7 @@ type ImportedAssembly =
         IsProviderGenerated: bool
         mutable TypeProviders: Tainted<ITypeProvider> list
 #endif
-        FSharpOptimizationData: Microsoft.FSharp.Control.Lazy<Option<Optimizer.LazyModuleInfo>>
+        FSharpOptimizationData: Microsoft.FSharp.Control.Lazy<Optimizer.LazyModuleInfo option>
```

Comment:
FWIW this should just be `Lazy` - not sure why the full namespace is quoted - can clean up separately